packages/office-ui-fabric-react/src/DragDrop.ts

```diff
@@ -0,0 +1 @@
+export * from 'office-ui-fabric-react/lib/DragDrop';
```

Comment:
This is circular as written?

Reply:
hmmm that's odd. I might have copied the file in the wrong direction; will fix!

Reply:
resolved.